docs/sources/administration/correlations/create-a-new-correlation/index.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Create a new correlation
+weight: 40
+---
+
+# Create a new correlation
+
+## Before you begin
+
+Make sure you have permission to add new correlations. Only users with write permissions to data sources can define new correlations.
+
+## Create a correlation in Administration page
+
+1. Go to the Administration section in Grafana.
+1. Open Correlations page.
+1. Click the “Add” button in the top right corner.
+1. Provide a **label** for the correlation.
+1. Provide an optional **description**.
+1. Go to the next page.
+1. Provide **target data source**.
+1. Provide **target query** using variables.
+1. Go to the next page.
+1. Provide **source data source**.
+1. Provide **results field**.
+1. Add transformations if you need variables that are not fields in the source data source.
+1. Click “Add” to add a new transformation.
+1. Select the type of a transformation.
+1. Configure transformation depending on the selected type.
+1. Save correlation.
+
+You can edit correlation in the same way, but you cannot change the selected data sources.
+
+## Create a correlation with provisioning
+
+Provision correlations by extending provisioned data sources. Correlations are defined as a subsection of the source data source configuration:
+
+```yaml
+datasources:
+  - name: Data source name # source data source
+    ...
+    jsonData:
+    ...
+    correlations:
+      - targetUID: uid
+        label: "test"
+        description: "..."
+        config:
+          type: "query"
+          target:
+            expr: "..."
+          field: "name"
+          transformations:
+            - type: regex
+              field: "test"
+              expression: /\w+/
+              mapValue: "other"
+            - type: logfmt
+              field: "test"
+```
+
+Description of provisioning properties:
+
+**targetUID**
+: Target data source UID
+
+**label**
+: Link label
+
+**description**
+: Optional description
+
+**config**
+: Config object
+
+**config.type**
+: Correlation type. “query” is the only supported type at the moment
+
+**config.target**
+: [Target query model]({{< relref "#determine-target-query-model-structure" >}})
+
+**config.field**
+: Name of the field where link is shown
+
+**config.transformations (list)**
+: List of transformation objects
+
+**transformation.type**
+: regex, or logfmt
+
+**transformation.field**
+: Input field for the transformation
```

Comment:
"The field that will be transformed. If this is not defined, it will apply the transformation to the data from the correlation's config.field."

Reply:
Sorry for not catching this on the first pass D:

Reply:
Thanks! No worries, happy to take as many takes as it needs :D

Reply:
Updated in 29260a9ba14aff32767962819f41c8a1641a9dc1